token_dispenser/token_dispenser_lambda.py

```diff
@@ -0,0 +1,175 @@
+from cryptography.hazmat.primitives.serialization.pkcs12 import load_key_and_certificates
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives.serialization import Encoding, PrivateFormat, NoEncryption
+from tempfile import NamedTemporaryFile
+from token_dispenser.aws.s3 import download_s3_file
+from token_dispenser.aws.secret_manager import get_secret_value
+from token_dispenser.aws.launchpad_token import get_token
+import token_dispenser.configuration as config
+from token_dispenser.repository.token_repo import put_token, get_token_by_client_id
+import os, json, re
+import logging
+from token_dispenser.logging_config import initialize_logger, shared_logger
+import time
+
+cached_cert_file:NamedTemporaryFile = None
+# Set the logging level dynamically
+log_level = getattr(logging, config.LOG_LEVEL, logging.INFO)
```

Comment:
Don't need the `logging.INFO` default since the default is set in the `token_dispenser.configuration` module